apps/web/src/app/support/confirmation/page.tsx

```diff
@@ -0,0 +1,74 @@
+import type { Stripe } from 'stripe';
+import { stripe } from '../_utils/stripe';
+import { auth } from '~/server/auth';
+import { prisma } from '@repo/db';
+import { Prisma } from '@repo/db/types';
+import dynamic from 'next/dynamic';
+
+const ContentNoSSR = dynamic(() => import('./content'), {
+  ssr: false,
+});
+
+export default async function ResultPage({
+  searchParams,
+}: {
+  searchParams: { session_id: string };
+}) {
+  if (!searchParams.session_id)
+    throw new Error('Please provide a valid session_id (`cs_test_...`)');
+
+  const session = await auth();
+
+  const checkoutSession: Stripe.Checkout.Session = await stripe.checkout.sessions.retrieve(
+    searchParams.session_id,
+    {
+      expand: ['line_items', 'payment_intent'],
+    },
+  );
+
+  const paymentIntent = checkoutSession.payment_intent as Stripe.PaymentIntent;
+
+  if (
+    session &&
+    paymentIntent.status === 'succeeded' &&
+    checkoutSession.amount_total &&
+    checkoutSession.amount_total > 0
+  ) {
+    const amountTotal = (checkoutSession.amount_total / 100).toFixed(2);
```

Comment:
What is the value returned from the stripe API here?

Just curious why we have to do mafs.

Reply:
its just a whole number. i think its fine either way

so 5.50 would be 550